gradio/components.py

```diff
@@ -273,12 +273,13 @@ def as_example(self, input_data):
 
 
 class FormComponent:
-    expected_parent = Form
+    def get_expected_parent(self) -> Type[Form]:
+        return Form
 
 
 @document("change", "submit", "blur", "style")
 class Textbox(
-    Changeable, Submittable, Blurrable, IOComponent, SimpleSerializable, FormComponent
+    FormComponent, Changeable, Submittable, Blurrable, IOComponent, SimpleSerializable
```

Comment:
Why did we need to change this order?

Reply:
Ah so I made `get_expected_parent()` a method that is defined both in the class `Block` and in the class `FormComponent`. For the one in `FormComponent` to override the one in `Block`, it has to come before `IOComponent`.

Reply:
I don't love this btw because it is kind of error-prone but I'll look for a way to improve when I work on `components.py`